### ISSUE TITLE
Orange LED flashing after unsuccessful hack / no MiHome connection possible

### ISSUE THREAD

**Issue by mikklaus** [samtap/fang-hacks#158]
Hey everyone, 
Went through the list of solved / open issues about this hack, but didn't find anything similar to my case yet. My story started 2 weeks ago when I went sick of MiHome Chinese menus and "can be used only in Mainland China" message. So, find this feed with the hack and decided to apply. Not from the first try, but the hack passed and I've got the script working, all network services up and running, but the http://DEVICE-IP/image.jpg showed 404 error page. Tried everything by stopping and starting services with web interface of the hack. Definitely left it working for one day, and coming back home in the evening I've seen camera was no longer connected to the network. After that never succeeded to restart it by powering off and on - never worked. Device always appeared offline in MiHome app, and pairing it again to the Mi Account never worked again - after the first "bla-bla" in Chinese and filling in the wi-fi connection, camera freezed when scanning the QR-code (led stopped flashing) and only powering off could solve. 
After that, I've tried to pass the hack again. There is a "blin-blin" sound, but site is never up. From router, the camera was undetectable, so I guessed that's because wi-fi information was last by camera, but as soon as I could set it through MiHome app, I've put on the SD card the little code to set Wi-Fi connection setting. 

Long story short... right now I'm stuck on LED constantly flashing orange, so that I can not even do the reset to factory firmware as it requires LED to be flashing blue, but not orange.

Question : does anyone experienced the same and find a way out?

**Comment by samtap** [comment]
I'm sorry it doesn't work for you but I can't keep answering these kind of 'issues'. If you need support, join discord.
